schemas/input/csl-data.json

```
@@ -463,10 +465,27 @@
               "type": ["string", "number", "boolean"]
             },
             "literal": {
+              "title": "Literal name text; should not be parsed",
+              "description": "Use for institutional creator names; e.g. 'National Institutes of Health'",
+              "type": "string"
+            },
+            "literal-short": {
+              "title": "Short form of literal",
+              "description": "Use for short institutional creator names; e.g. 'NIH'",
+              "type": "string"
+            },
+            "alternate": {
+              "title": "Alternative name, such as screen name for online item or real name of pseudonymous author.",
+              "description": "E.g. rendered as 'Smith, J. [@JSmith]'",
               "type": "string"
             },
             "parse-names": {
               "type": ["string", "number", "boolean"]
+            },
+            "is-uncertain": {
```

```suggestion
            "uncertain": {
```

To align with new date model.